modules/beamdyn/src/BeamDyn.f90

```diff
@@ -1966,19 +1966,17 @@ SUBROUTINE BD_CalcOutput( t, u, p, x, xd, z, OtherState, y, m, ErrStat, ErrMsg,
    INTEGER(IntKi)                               :: ErrStat2                     ! Temporary Error status
    CHARACTER(ErrMsgLen)                         :: ErrMsg2                      ! Temporary Error message
    CHARACTER(*), PARAMETER                      :: RoutineName = 'BD_CalcOutput'
-   LOGICAL                                      :: CalcWriteOutput
+   LOGICAL                                      :: IsFullLin
```

Comment:
I'm not sure I follow why the name was changed here.  Is full linearization required for the outputs?

Reply:
I think I follow now.  The `dYdu` term needs this.